python/opae.admin/opae/admin/tools/fpgasupdate.py

```diff
@@ -52,6 +52,9 @@
 from opae.admin.fpga import fpga
 from opae.admin.utils.progress import progress
 from opae.admin.version import pretty_version
+import pacsign
+from pacsign import database
```

Comment:
Probably don't want to insert a reference to pacsign in fpgasupdate, because they are two distinct packages; and there has been some talk of removing pacsign from the opae-sdk repo.  If all you need is database.CONTENT_FACTORY, then just create a CONTENT_FACTORY variable in fpgasupdate to avoid the dependency.

Reply:
Thanks, I have implemented this.